src/system.c

```diff
@@ -1462,7 +1462,20 @@ Int8 SyNanosecondsSinceEpoch()
 {
   Int8 res;
 
-#if defined(HAVE_CLOCK_GETTIME) && defined(CLOCK_MONOTONIC)
+#if defined(SYS_IS_DARWIN)
+  UInt8 nanosecs;
+
```

Comment:
Nope, no good reason. I wrote the code separately, and did a poor job of merging it in.